skimage/filters/thresholding.py

```diff
@@ -384,3 +389,91 @@ def threshold_li(image):
             new_thresh = temp + tolerance
 
     return threshold + immin
+
+
+def threshold_minimum(image, nbins=256, bias='min'):
+    """Return threshold value based on minimum method.
+
+    A histogram is computed and smoothed until there are only two maximums.
+    Then the minimum between these is found.
+
+    Parameters
+    ----------
+    image : array
+        Input image.
+    nbins : int, optional
+        Number of bins used to calculate histogram. This value is ignored for
+        integer arrays.
+    bias : {'min, 'mid', 'max'}, optional
+        'min', 'mid', 'max' return lowest, middle, or highest pixel value
+        with minimum histogram value.
+
+    Returns
+    -------
+    threshold : float
+        Computed threshold value.
+        May return 0 if the algorithm fails.
```

Comment:
I'm not sure to understand this statement. what fail means? The only case I can imagine is if the first maxima is at 0 in the histogram. I don't think that failure is the correct wording. To me, it would show an inappropriate choice of algorithm.

I vote to remove this statement.